.github/workflows/ci.yml

```diff
@@ -15,30 +15,8 @@ env:
   PYTEST_ADDOPTS: --exitfirst
 
 jobs:
-  metadata:
-    name: Metadata
-    runs-on: ubuntu-latest
-    outputs:
-      python_headline_version: ${{ steps.list-python-versions.outputs.python_headline_version }}
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.9
-
-      - name: List Python versions
-        id: list-python-versions
-        run: |
-          pip install hdev
-          echo "python_headline_version=`hdev python_version --style plain --first`" >> $GITHUB_OUTPUT
```

Comment:
Nice! We can remove the entire metadata job. This will save time. I'm glad to see the custom concept of a "headline version" gone too